kitty/line.c

```diff
@@ -235,35 +235,35 @@ line_as_ansi(Line *self, Py_UCS4 *buf, index_type buflen) {
     if (limit == 0) return 0;
     char_type previous_width = 0;
 
-    WRITE_SGR("0");
```

Comment:
Forgot this one -- I cannot think of any case where resetting the mode is needed at start of line. In particular, I had a patch that carried the sgr mode from the previous line, until I noticed less completely ignores this and resets the mode to zero at the start of every line...

I guess that could have been done for compat with other pagers, or other versions of less? I'll remove that commit and the equivalent in the rewrap function if you think that is useful.